satpy/etc/areas.yaml

```diff
@@ -166,6 +166,108 @@ msg_seviri_iodc_9km:
     lower_left_xy: [-5567248.074173927, -5567248.074173927]
     upper_right_xy: [5567248.074173927, 5567248.074173927]
 
+msg_seviri_fes_9km_ext:
```

Comment:
Do these match what was discussed in the other issues about "standard" area definitions?

Reply:
It took a while to get back to this PR...

Yes indeed. We had an internal discussion on this topic and after that I did some further harmonization to have the segmented area definitions to better match the corresponding pixel-level area definitions. Following that discussion I also spotted two bugs in the area extents of the new area definitions. These bugs have now been resolved.